### DISCUSSION TITLE
Renault Talisman-2016 passenger seat easy access opening DDT4ALL

### DISCUSSION THREAD

**Discussion by engindalmis** [cedricp/ddt4all#1096] [Q&A]
Hello;
How can I make easy access to the passenger seat in my Talisman vehicle with dd4all?
Thank you now.

**Comment by matzel82** [comment]
What ??

**Comment by engindalmis** [comment]
![msedge_9wycyfBqcC](https://github.com/cedricp/ddt4all/assets/46320921/9f93c69d-e3ec-4d3b-aca7-4b17f4f03ac1)

**Comment by engindalmis** [comment]
![msedge_9wycyfBqcC](https://github.com/cedricp/ddt4all/assets/46320921/71f09de1-4fe2-4dc0-8d7f-9cbf32527cfe)

**Reply by hakandroid** [reply]
başarabildiniz mi hocam bu işlemi? bende de menüler çıkıyor ama on konumuna getirdikten 10 sn sonra tekrar eski haline geliyor.

**Comment by matzel82** [comment]
Im only speaking English and German

**Reply by hakandroid** [reply]
Hello, I have a Renault Talisman car. I haven't been able to make the passenger seat easy to access and control from the driver's side. The menus appear, but when I try to start it, it resets again. Have you done this?
